generator/api/src/main/java/io/fabric8/maven/generator/api/support/BaseGenerator.java

```diff
@@ -121,8 +122,8 @@ protected String getAlias() {
     }
 
     protected boolean shouldAddDefaultImage(List<ImageConfiguration> configs) {
-        String merge =  getConfigWithSystemFallbackAndDefault(Config.alias, "fabric8.generator.alias", "false");
-        return !containsBuildConfiguration(configs) || Boolean.parseBoolean(merge);
+        String merge =  getConfigWithSystemFallbackAndDefault(Config.alias, "fabric8.generator.alias", "");
+        return !containsBuildConfiguration(configs) || Strings.isNotBlank(merge);
     }
```

Comment:
thanks, looks better. And now I get it, too ;-) Being explicit makes much more sense. There are still some spots which need to be moved into the generators / enrichers, will tackle that this week (e.g. #223).

thx ;-)